test/functional/core/job_spec.lua

```diff
@@ -179,28 +189,47 @@ describe('jobs', function()
   end)
 
   it("will not buffer data if it doesn't end in newlines", function()
-    if helpers.pending_win32(pending) then return end  -- TODO: Need `cat`.
     if os.getenv("TRAVIS") and os.getenv("CC") == "gcc-4.9"
       and helpers.os_name() == "osx" then
       -- XXX: Hangs Travis macOS since e9061117a5b8f195c3f26a5cb94e18ddd7752d86.
       pending("[Hangs on Travis macOS. #5002]", function() end)
       return
     end
 
-    nvim('command', "let j = jobstart(['cat', '-'], g:job_opts)")
+    nvim('command', "let j = jobstart(has('win32') ? ['find', '/v', ''] : ['cat', '-'], g:job_opts)")
     nvim('command', 'call jobsend(j, "abc\\nxyz")')
-    eq({'notification', 'stdout', {0, {'abc', 'xyz'}}}, next_msg())
+    eq({'notification', 'stdout', {0, iswin() and {'abc', ''} or {'abc', 'xyz'}}}, next_msg())
```

Comment:
Probably should used expect_msg_seq

Reply:
For this line only? This is expected behaviour for `find.exe` the string has to end with newline (or some character for find.exe receive the input). As for the stdout after jobstop, I don't know why.

I can experiment with `expect_msg_seq` for both stdout.

Reply:
On 2nd thought, I'll revert it. Leave it for `cat` when the situation improves. There's enough tests for find.exe anyway.